src/transformers/pipelines.py

```diff
@@ -2734,12 +2734,17 @@ def pipeline(
             identifier or an actual pretrained model configuration inheriting from
             :class:`~transformers.PretrainedConfig`.
 
-            If not provided, the default for the :obj:`task` will be loaded.
+            If not provided, the default configuration file for the requested model will be used: either the given 
+            :obj:`model`'s configuration, or - if no :obj:`model` was given - this :obj:`task`'s default model's 
```

Comment:
I'd prefer to have two sentences here instead of the part between em-dashes (like you did for the tokenizer). I think it would be clearer this way.

Reply:
Done!